extensions/amp-analytics/amp-analytics.md

```diff
@@ -685,6 +685,105 @@ Referrer policy is only available for `image` transport. If `referrerPolicy: no-
+`https://www.amp.example.com/foo` | yes | matches source hostname
+`https://www.foo.example.com/bar` | no | not the same hostname
+`https://www.google.com` | no | different domain
+
+Remember that this is just the default behavior if no `destinationDomains` array is specified. You may always add addition domains to match in this array.
+
+
+##### Configuration Inheritance
+As outlined above some items in the configuration objects will have default values. Additionally, you may override these values at the `amp-analytics` element level to suit your use case. Then values will then be merged where the more specific configuration will take precedence over their parents. Keep in mind these defaults are set only for the `linkers` object in which they are defined. If you have multiple `amp-analytics` elements on your page you will need to set the configuration for each top level linkers object you wish to enable.
+
+Example:
+```javascript
+"linkers": {
+  "enabled": true, // This enables all child linkers contained in this object.
+  "proxyOnly" : false,
+  "linker1" : {
+    "ids" : {
+      "cid": "CLIENT_ID(_ga)",
+    }
+  },
+  "linker2" : {
+    "proxyOnly": true
+    "ids" : {
+      "gid": "CLIENT_ID(foo)",
+    }
+}
+```
+
+In this example `linker2` would be appended only on the proxy, while `linker1` would be shown in all contexts as defined in the parent config.
+
+##### Checksum
+The checksum is a unique value generated by the linker that can be used to prevent accidental Id sharing. Publishers and/or anayltics vendors can generate this checksum on the destination page and ensure that it matches. Since this checksum contains a time input, a common practice is to check it against a few values generated by adding or subtracting a time tolerance.
+
+It is generated from the following formula.
+`base36(CRC32(fingerprint*timestampRoundedInMin*key1*val1*key2*val2...))`
+
+- `fingerprint` - A semi-unique value for page visitors. Generated by using `User Agent + * + timezone + * + language`. [code](https://github.com/ampproject/amphtml/blob/d8b40f62bbfe217307a21e600633343d9938aa45/extensions/amp-analytics/0.1/linker.js#L67)
+- `minutesSinceEpoch` - Time since epoch. Floored to minute resolution. [code](https://github.com/ampproject/amphtml/blob/d8b40f62bbfe217307a21e600633343d9938aa45/extensions/amp-analytics/0.1/linker.js#L103)
+- `key1...` - Each key value pair that will be passed along. The key will remain in its raw form, while the value will be url-safe base64 endcoded. [code](https://github.com/ampproject/amphtml/blob/d8b40f62bbfe217307a21e600633343d9938aa45/extensions/amp-analytics/0.1/linker.js#L113)
+- `CRC32` - A standard CRC32 implementation. [code](https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/0.1/crc32.js)
+- `base36` - The result of the CRC32 algorithm is base36 encoded. [code](https://github.com/ampproject/amphtml/blob/d8b40f62bbfe217307a21e600633343d9938aa45/extensions/amp-analytics/0.1/linker.js#L62)
```

Comment:
@lannka I noticed the code still does this. I thought there was a conversation about making everything base64?

Reply:
this was not changed, due to the complexity of change number to bytes then base64.